client/src/game/reducers/statsReducer.ts

```diff
@@ -54,15 +56,52 @@ function statsReducerFunction(
         // similar to what happens when the team gets a strike
         stats.potentialCluesLost += clueTokensRules.discardValue(variant);
       }
+      stats.doubleDiscard = false;
 
       break;
     }
 
-    default: {
+    case "discard": {
+      // Handle double discard calculation
```

Comment:
isn't this a bug because double discard will never be set to false unless two people discard in a row

Reply:
notice the other places in the code where `stats.doubleDiscard = false` is written: lines 32 and 59

Reply:
although I'm not sure if this accounts for dda situations caused by a misplay. the previous code didn't seem to either and yet I've seen it work. Is that because a misplay is encoded as a "strike" action plus a "discard" action?

Reply:
no idea, strike action will be removed at some point so if it relies on a strike action then that is a bug

Reply:
the only thing here that relies on strike action is `potentialCluesLost` :P